.github/workflows/cut-release.yml

```diff
@@ -0,0 +1,59 @@
+name: "cut-release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        required: false
+jobs:
+  cut_release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code 
+        uses: actions/checkout@v3
+      - name: "Build & test"
+        run: |
+          sudo apt-get install build-essential -y
+          ./autogen.sh
+          ./configure
+          make distcheck
+      - name: Update VERSION file on checkout 
+        run: |
+          set -x
+          sudo apt-get install -y python3-semver
+          if [ -z "$NEW_VERSION" ]; then
+             echo "Current Version: $(cat VERSION)"
+             pysemver bump patch $(cat VERSION) > VERSION
+          else
+             echo $NEW_VERSION > VERSION
+          fi
+          export NEW_VERSION=$(cat VERSION)
```

Comment:
It would be good to compare that new version is larger than older version here. You can use `pysemver compare` for that purpose, check the output, if it's -1 or 0 it means the first argument is smaller or they're equal, if it's 1 it means the second argument is the newer semver (which is what we want).